### DISCUSSION TITLE
steamwebhelper still not functioning

### DISCUSSION THREAD

**Discussion by rocketstrong600** [ptitSeb/box64#528] [Q&A]
I have installed box64 steam_chrome branch on a sdm845 platform and steam web helper is still not functioning.

**Comment by ptitSeb** [comment]
I'm aware. 
I just pushed a workaround. It seems Armbian behave differently than debian here, no sure why.

**Reply by rocketstrong600** [reply]
I will try the new commit. my setup is a bit different it's arch Linux arm with the mainline Linux kernel patched for OnePlus 6 postmarket os. I am then using a debian chroot for both arm64 and armhf libs.

**Reply by ptitSeb** [reply]
Use a `~/.box64rc` file to setup specific parameters for `[steamwebhelper]`, it's the easiest way.

**Reply by rocketstrong600** [reply]
doesn't the make install put a box64rc in /etc

**Reply by ptitSeb** [reply]
It does. But the one in your home is last load and take precedence. So if a profile exist on both, only the one in home will count.

**Reply by rocketstrong600** [reply]
I got it to work I remade the chroot and used systemd-nspawn instead.![PXL_20230226_161246907.jpg](https://user-images.githubusercontent.com/6792632/221422292-ae53824b-a577-435e-96f4-ac56d80c3372.jpg)

**Reply by ptitSeb** [reply]
Oh, nice! Full steam library on a phone :)
